scripts/core/superdesk-api.d.ts

```diff
@@ -523,7 +523,9 @@ declare module 'superdesk-api' {
     export interface IUserRole extends IBaseRestApiResponse {
         _id: string;
         name: string;
-        privileges: any;
+        privileges?: {
+            [privilege: string]: number;
```

Comment:
You can also use a number value as a type `[privilege: string]: 1;`. I think there are no other values.